example/minimal/main.go

```diff
@@ -29,15 +31,26 @@ func (s *Server) registerUser(schema *schemabuilder.Schema) {
 func (s *Server) registerQuery(schema *schemabuilder.Schema) {
 	object := schema.Query()
 
-	object.FieldFunc("users", func(ctx context.Context) ([]*User, error) {
+	var tmp MessageType
+	schema.Enum(tmp, map[string]interface{}{
+		"first":  MessageType(1),
+		"second": MessageType(2),
+		"third":  MessageType(3),
+	})
+
+	object.FieldFunc("users", func(ctx context.Context, args struct {
+		EnumField MessageType
```

Comment:
since this is the example code, let's make this more semantically interesting? maybe a `ROle` enum for `user` / `manager` / `administrator` or something